### ISSUE TITLE
Cannot access WebView content with flutter_accessibility_service

### ISSUE THREAD

**Issue by answer0732** [X-SLAYER/flutter_accessibility_service#38]
Hi, I want know is it possible to obtain the content after the WebView finishes rendering?

**Comment by nhCoder** [comment]
use like this

`<?xml version="1.0" encoding="utf-8"?>
<accessibility-service xmlns:android="http://schemas.android.com/apk/res/android"
                       android:description="@string/accessibility_service_description"
                       android:accessibilityEventTypes="typeViewClicked|typeViewFocused|typeViewTextChanged|typeWindowContentChanged|typeWindowStateChanged"
                       android:accessibilityFeedbackType="feedbackGeneric"
                       android:notificationTimeout="300"
                       android:accessibilityFlags="flagReportViewIds|flagRequestEnhancedWebAccessibility|flagRetrieveInteractiveWindows"
                       android:canRetrieveWindowContent="true">
</accessibility-service>`
